Marlin/Marlin_main.cpp

```diff
@@ -6456,11 +6148,11 @@ void process_next_command() {
 
       #if ENABLED(AUTO_BED_LEVELING_FEATURE) || ENABLED(MESH_BED_LEVELING)
         case 29: // Auto bed leveling
-          #if ENABLED(EXT_ADC) 
-            gcode_M237(); // M237: Auto bed leveling function using profilometer data
-          #else
+          // #if ENABLED(EXT_ADC) 
+          //   gcode_M237(); // M237: Auto bed leveling function using profilometer data
```

Comment:
Moved the contents of gcode_M237 to G29, to avoid confusion in the future when calling one code and having it activate another